extensions/amp-a4a/0.1/callout-vendors.js

```diff
@@ -48,10 +48,15 @@ export const RTC_VENDORS = {
     disableKeyAppend: true,
   },
   prebidrubicon: {
-    url: 'https://prebid-server.rubiconproject.com/openrtb2/amp?tag_id=REQUEST_ID',
+    url: 'https://prebid-server.rubiconproject.com/openrtb2/amp?tag_id=REQUEST_ID&w=ATTR(width)&h=ATTR(height)&ow=ATTR(data-override-width)&oh=ATTR(data-override-height)&ms=ATTR(data-multi-size)&slot=ATTR(data-slot)&tgt=TGT&curl=CANONICAL_URL&to=TIMEOUT&purl=HREF&adc=ADCID',
```

Comment:
I would consider putting ADCID before HREF, as we truncate if the URL is too long, and I assume you'd rather not lose ADCID